### ISSUE TITLE
<a11y>Element with "img" role has no label or an empty label

### ISSUE THREAD

**Issue by xubinnb** [carbon-design-system/ibm-products#4033]
===== ENVIRONMENT =====
Browser: Version 105.0.5195.125 (Official Build) (arm64)
Automatically update Chrome for all users Learn more
Test Tool Type: IBM Accessbility Checker

===== DEFECT DESCRIPTION ===== 
Problem 1: Element with "img" role has no label or an empty label

=====STEPS TO REPRODUCE =====
Go to https://dev.console.test.cloud.ibm.com/projects

Use IBM Equal Access Accessibility Checker to scan

===== EXPECTED RESULT =====
[IBM 1.1.1 Non-text Content](https://www.ibm.com/able/requirements/requirements/#1_1_1)

All non-text content that is presented to the user has a text alternative that serves the equivalent purpose.

===== SCREEN CAPTURE =====
<img width="1276" alt="image" src="https://media.github.ibm.com/user/42629/files/57acec22-33ad-4f7b-919f-c9443c801129">

**Comment by elycheea** [comment]
@xubinnb Can you share what version of our library you are using? I believe @davidmenendez fixed a related issue previously so it’ll help to know if this is a new issue or regression. Thank you!

**Comment by xubinnb** [comment]
@elycheea As Flannan(flannanl@ca.ibm.com)) mentioned, It seems they fix this in their v2 version. But since we are using v1, we need them to port the fix to v1. Thanks.
